plugins/visual/visualqml/input/qml/PathInputBox.qml

```diff
@@ -52,8 +52,8 @@ Item{
 
     InputBox{
         id: pathInput
-        anchors.left: parent.left
-        width: parent.width - 31
+        anchors.left: parent ? parent.left : undefined
```

Comment:
`parent` checks within declarative hierarchies are not necessary. If the box signals an error, it must be from somewhere else where this component was dynamically allocated